lib/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -174,6 +174,60 @@ a {
 
 ## Optional secondary options
 
+### `ignoreLonghands: ["string"]`
```

Comment:
[suggest] I feel supporting also regexes makes sense. Any thoughts?

```suggestion
### `ignoreLonghands: ["/regex/", /regex/, "string"]`
```

Reply:
I don't think so for 2 reasons:

- it will complicate the implementation
- I couldn't find a use case for regexes

Reply:
I see. Let's leave it as-is. We can revisit later if people request.